038-optimization-proposal-autoapproval.md

```diff
@@ -0,0 +1,66 @@
+# Auto-approval mechanism for optimization proposals
+
+This proposal is about allowing the user to create a `KafkaRebalance` custom resource for getting an optimization proposal and running a rebalance, but without the need of the manual approval via the `strimzi.io/rebalance=approve` annotation.
+It means that the user can just create the `KafkaRebalance` custom resource and the corresponding generated optimization proposal will be automatically approved.
+
+## Current situation
+
+Currently, when the user wants to run a cluster rebalance, he has to create a `KafkaRebalance` custom resource for getting an optimization proposal first.
```

Comment:
Maybe this way it is better to avoid any he/she discussions ;-)

```suggestion
Currently, when the users want to run a cluster rebalance, they have to create a `KafkaRebalance` custom resource for getting an optimization proposal first.
```